lib/PlaidClient.js

```diff
@@ -427,7 +427,21 @@ function(access_token, webhook_code, cb) {
       access_token: access_token,
       webhook_code: webhook_code,
     },
-  }, cb, true);
```

Comment:
Why was this line changed?

Reply:
I don't believe it was.  The three lines before and after changes are shown for context within a diff.

Reply:
I have pushed since then but it has actually changed. the boolean parameter is ignored if the cb parameter passed in is a function. it is not good but unfortunately this is how it is. so removing the true is a noop.